support/map.h

```diff
@@ -2119,18 +2204,21 @@ int Mono_Posix_Syscall_getgrnam_r (const char* name, struct Mono_Posix_Syscall__
 gint64 Mono_Posix_Syscall_gethostid (void);
 int Mono_Posix_Syscall_gethostname (char* name, guint64 len);
 int Mono_Posix_Syscall_getlogin_r (char* name, guint64 bufsize);
+int Mono_Posix_Syscall_getpeername (int socket, struct Mono_Posix_Sockaddr* address);
 int Mono_Posix_Syscall_getpwent (struct Mono_Posix_Syscall__Passwd* pwbuf);
 int Mono_Posix_Syscall_getpwnam (const char* name, struct Mono_Posix_Syscall__Passwd* passwd);
 int Mono_Posix_Syscall_getpwnam_r (const char* name, struct Mono_Posix_Syscall__Passwd* pwbuf, void** pwbufp);
 int Mono_Posix_Syscall_getpwuid (unsigned int uid, struct Mono_Posix_Syscall__Passwd* passwd);
 int Mono_Posix_Syscall_getpwuid_r (unsigned int uid, struct Mono_Posix_Syscall__Passwd* pwbuf, void** pwbufp);
+int Mono_Posix_Syscall_getsockname (int socket, struct Mono_Posix_Sockaddr* address);
 int Mono_Posix_Syscall_getsockopt (int socket, int level, int option_name, void* option_value, gint64* option_len);
 int Mono_Posix_Syscall_getsockopt_linger (int socket, int level, int option_name, struct Mono_Posix_Linger* option_value);
 int Mono_Posix_Syscall_getsockopt_timeval (int socket, int level, int option_name, struct Mono_Posix_Timeval* option_value);
 int Mono_Posix_Syscall_gettimeofday (struct Mono_Posix_Timeval* tv, void* ignore);
 gint64 Mono_Posix_Syscall_getxattr (const char* path, const char* name, unsigned char* value, guint64 size);
-int Mono_Posix_Syscall_L_ctermid (void);
```

Comment:
This might be due to https://github.com/mono/mono-tools/commit/9de0fe06a99003462df3c77dda93718ab85da487 which changed from CompareOptions.Ordinal | CompareOptions.IgnoreCase to CompareOptions.OrdinalIgnoreCase (what changed is how '_' is ordered compared to letters):

    System.Globalization.CultureInfo.InvariantCulture.CompareInfo.Compare ("_", "a", System.Globalization.CompareOptions.OrdinalIgnoreCase) = 30
    System.Globalization.CultureInfo.InvariantCulture.CompareInfo.Compare ("_", "a", System.Globalization.CompareOptions.Ordinal | System.Globalization.CompareOptions.IgnoreCase) = -1 (but this is no longer supported in newer mono versions)